xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -907,9 +907,9 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The object to add to the end of the collection.</param>
+        <summary>Adds an object to the end of the collection.</summary>
+        <returns>The index at which the value has been added.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.IList> interface.  

           ]]></format>
         </remarks>
```